packages/api-gateway/src/drivers/http/adapters/index.js

```diff
@@ -1,9 +1,17 @@
-'use strict';
-
+// Internal dependencies
 const mathAdapters = require('./math/handlers');
+const geolocationAdapters = require('./geolocation/handlers');
 const statusAdapters = require('./status/handlers');
+const adminPanelAdapters = require('./admin_panel/handlers');
+const notificationsAdapters = require('./notifications/handlers');
+const placesAdapters = require('./places/handlers');
+const bookingAdapters = require('./booking/handlers');
 
 module.exports = {
-	mathAdapters,
-	statusAdapters,
-}
+  mathAdapters,
+  statusAdapters,
+  adminPanelAdapters,
+  notificationsAdapters,
+  placesAdapters,
+  bookingAdapters,
```

Comment:
Looks like you removed accidentally the geolocationAdapters from the exported modules, Can you added again?
```suggestion
  bookingAdapters,
  geolocationAdapters,
```